python/lsst/pipe/base/argumentParser.py

```diff
@@ -81,19 +83,40 @@ class DataIdContainer(object):
 
     Parameters
     ----------
-    level
-        Unknown.
+    level : `str`
+        The lowest hierarchy level to descend to for this dataset type,
+        for example `"amp"` for `"raw"` or `"ccd"` for `"calexp"`.
+        Use `""` to use the mapper's default for the dataset type.
+        This class does not support `None`, but if it did, `None`
+        would mean the level should not be restricted.
 
     Notes
     -----
-    Override for data IDs that require special handling to be converted to ``data references``, and specify
-    the override class as ``ContainerClass`` for ``add_id_argument``. (If you don't want the argument parser
-    to compute data references, you may use this class and specify ``doMakeDataRefList=False`` in
-    ``add_id_argument``.)
+    **Fields:**
```

Comment:
Should these be documented in a `Parameters` block, or should they be declared as class members and documented there?

https://developer.lsst.io/python/numpydoc.html#examples-of-constant-and-class-attribute-docstrings